admin_manual/configuration/server/config_sample_php_parameters.rst

```diff
@@ -197,11 +206,29 @@ This example is from a new ownCloud installation using MariaDB::
     'dbuser' => 'oc_carla',
     'dbpassword' => '67336bcdf7630dd80b2b81a413d07',
     'installed' => true,
+    'operation.mode' => 'single-instance',
   );
 
 .. Generated content below. Don't change this.
 .. ALL_OTHER_SECTIONS_START
 
+.. _mode_of_operation_label:
+
+Mode of Operation 
+-----------------
+
+ownCloud can operate as either a single instance or as a clustered instance. 
+This operations mode has various impacts on ownCloud's behavior.
+
+A single instance, the default, means that ownCloud is running on a single node (likely, the most common operating mode). 
+A clustered instance means that ownCloud is running on at least 2 nodes, if not more. 
+
+In single instance mode, all of the standard functionality, including *installing*, *updating*, and *removing* apps via the Marketplace are fully-supported.
+In clustered instance mode, *installing*, *updating*, and *removing* apps via the Marketplace **is not** supported. 
```

Comment:
Actually this should say that installing... via the **Market App** is not supported. You can still manually grab apps from the marketplace and put them to the apps folder.